plugins/azure/blobservice/blobContainersCmkEncrypted.js

```diff
@@ -0,0 +1,86 @@
+        async.each(locations.storageAccounts, (location, rcb) => {
+            const storageAccounts = helpers.addSource(
+                cache, source, ['storageAccounts', 'list', location]
+            );
+
+            if (!storageAccounts) return rcb();
+
+            if (storageAccounts.err || !storageAccounts.data) {
+                helpers.addResult(results, 3,
+                    'Unable to query for Storage Accounts: ' + helpers.addError(storageAccounts), location);
+                return rcb();
+            }
+
+            if (!storageAccounts.data.length) {
+                helpers.addResult(
+                    results, 0, 'No existing Storage Accounts found', location);
+                return rcb();
+            }
+
+            for (var storageAccount of storageAccounts.data) {
+                const blobContainers = helpers.addSource(
+                    cache, source, ['blobContainers', 'list', location, storageAccount.id]
+                );
+
+                if (!blobContainers || blobContainers.err || !blobContainers.data) {
+                    helpers.addResult(results, 3,
+                        'Unable to query Blob Containers: ' + helpers.addError(blobContainers),
+                        location, storageAccount.id);
+                } else if (!blobContainers.data.length) {
+                    helpers.addResult(results, 0, 'Storage Account does not contain blob containers', location, storageAccount.id);
+                } else {
+                    const encryptionScopes = helpers.addSource(
+                        cache, source, ['encryptionScopes', 'listByStorageAccounts', location, storageAccount.id]);
+
+                    if (!encryptionScopes || encryptionScopes.err || !encryptionScopes.data) {
+                        helpers.addResult(results, 3,
+                            'Unable to query encryption scopes for Storage Accounts: ' + helpers.addError(encryptionScopes),
+                            location, storageAccount.id);
+                        break;
+                    } else {
+                        var cmkEncryptionScopes = encryptionScopes.data.filter(function(scope) {
+                            return scope.keyVaultProperties && scope.keyVaultProperties.keyUri;
+                        }).map(function(scope) {
+                            return scope.name;
+                        });
+                        var unencryptedCmkBlobs = blobContainers.data.filter(function(blob) {
+                            return !cmkEncryptionScopes.includes(blob.defaultEncryptionScope);
+                        }).map(function(blob){
+                            return blob.name;
+                        });
+                        if (unencryptedCmkBlobs && unencryptedCmkBlobs.length){
+                            helpers.addResult(results, 2, `Storage Account does not have CMK encryption enabled on following blob containers: ${unencryptedCmkBlobs.join(',')}`, location, storageAccount.id);
```

Comment:
```suggestion
                            helpers.addResult(results, 2, `Storage Account does not have CMK encryption enabled on following blob containers: ${unencryptedCmkBlobs.join(',')}`, location, storageAccount.id);
```
the resource should be the blob container itself and not the storage account